styles/mobile-about.css

```diff
@@ -0,0 +1,333 @@
+  font-size: 32px;
+  line-height: 10px;
+  font-weight: 400;
+  color: #ed5342;
+}
+
+.headline-title h1 {
+  font-family: var(--font-family);
+  font-size: 50px;
+  font-weight: 700;
+  line-height: 50px;
+  color: #f53f2b;
+  margin-top: 0;
+  text-align: center;
+}
+
+.headline-paragraph {
+  font-family: var(--font-family);
+  font-size: 17px;
+  color: rgb(58, 58, 72);
+  font-weight: 500;
+  line-height: 27px;
+  border: 5px solid white;
+  padding: 0 0 0 10px;
+  text-align: center;
+  background-color: #fff;
+  padding-right: 10px;
+}
+
+.headline-time p {
+  font-family: var(--font-family);
+  font-size: 17px;
+  color: rgb(58, 58, 72);
+  font-weight: 500;
+  line-height: 27px;
+  text-align: center;
+  margin-bottom: 0;
+  margin-top: 30px;
+}
+
+.headline-venue h5 {
+  font-family: var(--font-family);
+  font-size: 17px;
+  color: rgb(58, 58, 72);
+  font-weight: 600;
+  margin: 0;
+  text-align: center;
+  margin-bottom: 2px;
+}
+
+.about-logo {
+  background-color: #fff;
+  padding: 20px 19px 25px 20px;
+  margin-bottom: 5px;
+}
+
+.about-logo-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.line {
+  width: 40px;
+  height: 2px;
+  display: flex;
+  background: #ed5342;
+}
+
+.about-logo-title h2 {
+  font-family: var(--font-family);
+  color: rgb(58, 58, 72);
+  text-align: center;
+  line-height: 48px;
+  font-size: 40px;
+  font-weight: 500;
+  margin-bottom: 0;
+}
+
+.about-logo-paragraph p {
+  font-family: var(--font-family);
+  font-size: 12px;
+  color: rgb(58, 58, 72);
+  font-weight: 500;
+  line-height: 27px;
+  padding: 0 0 0 10px;
+  text-align: center;
+  padding-right: 10px;
+}
+
+.about-logo-image img {
+  border: 2px solid rgb(216 216 227);
+  margin-bottom: 0;
+}
+
+.stories {
+  background-color: #fff;
+  padding-top: 19px;
+}
+
+.stories-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.story-title h2 {
+  font-family: var(--font-family);
+  font-size: 25px;
+  color: rgb(58, 58, 72);
+  font-weight: 500;
+  line-height: 27px;
+  margin-bottom: 0;
+}
+
+.desk-about-logo-title {
+  display: none;
+}
+
+/* .story-paragraph p {
+    font-family: var(--font-family);
+font-size: 16px;
+color: rgb(58, 58, 72);
+font-weight: 500;
+line-height: 27px;
+}
+
+.story-images{
+    padding: 20%;
+    background-image: url(../images/conference.png);
+    max-width: 100%;
+    margin-bottom: 32px;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.story-images2{
+    padding: 20%;
+    background-image: url(../images/conference.png) ;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+}  */
+
+/* .year{
+    text-align: center;
+    font-family: var(--font-family);
+    font-weight: 600;
+    color: #fff;
+    font-size: 30px;
+    font-weight: 600;
+    margin-bottom: -12px;
+}
+
+.para{
+    text-align: center;
+    font-family: var(--font-family);
+    font-weight: 400;
+    color: #fff;
+} */
+
```

Comment:
- [ ] Removing commented code not only makes it harder to find later, but it also makes it so people in the future don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. Please remove all commented code and unnecessary files from your code base :smile: